pkg/kubectl/explain/explain.go

```diff
@@ -63,6 +64,9 @@ func PrintModelDescription(fieldsPath []string, w io.Writer, schema proto.Schema
 	if err != nil {
 		return err
 	}
+	if schema == nil {
```

Comment:
the four-states-return-value-but-maybe-two? in go is quite terrible imo. I suspect it's a bug that we get a nil schema without an error?

Reply:
what is the situation that triggered the panic?

Reply:
I was considering that approach a viable fix, I'll look what's possible. This was found in https://github.com/kubernetes/kubernetes/pull/71192#discussion_r254784547 when we don't have an up-to-date information about CRD.

Reply:
The problem is that this fix hides the actual problem. I'd rather not implement it this way. I had a discussion with @roycaihw and we'll try to improve the empty object instead.

Reply:
I don't think it hides it… eero ring is better than panicking here, right?

Reply:
Well, I might be wrong, but I still don't think it should happen. In which case I'm in general in favor of panics, because they tend to force people to do the right thing.

Reply:
I prefer not to have panics, honestly ;)

Reply:
> I prefer not to have panics, honestly ;)

agree

Reply:
Let me rephrase. I don't want a panic, I just don't think we should have a nil schema in the first place.